docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -13,153 +13,235 @@ sidebar_position: 2
 
 ## Introduction
 
-Once deployed on a network, smart contracts cannot be changed. Their storage and logic **are set in stone**. If a smart contract is deployed with specific state variables and functions, these data and functionalities are there forever and cannot be altered.
+Externally Owned Accounts (EOAs) are the basic type of **account** in Ethereum controlled by a private key. If the private key is **compromised**, anyone can execute transactions from the account and access any assets it holds. As well, EOAs have no built-in mechanism for attaching any information or data, making it difficult to identify the person or entity using the account. They can only perform simple interactions using the **[CALL](https://www.evm.codes/#f1)** opcode and creating contracts using the **[CREATE](https://www.evm.codes/#f0)** opcode.
 
-The **[ERC725 Standard](https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md)** proposed in 2017 describes a generic key-value store and generic execution function that could be used as the basis for an account. A blockchain-based account can then be created by combining ERC725 with the interactivity and signature verification mechanism from the **[LSP1-UniversalReceiver Standard](../generic-standards/lsp1-universal-receiver.md)** and **[ERC1271 Standard](https://eips.ethereum.org/EIPS/eip-1271)**.
+These issues can be addressed by the **[ERC725](../lsp-background/erc725.md)** standard, which provides more operations to execute and a flexible way to attach data for the contract even after it's been deployed.
 
-![LSP0 ERC725 Account overview](/img/standards/lsp0/lsp0-erc725account-overview.jpeg)
+However, for a smart contract-based account to be viable in the long term, it should have much more functionalities than the ability to execute and to attach data. The features that makes a smart contract an account are: the ability to **verify signed messages**, be **notified of incoming tokens**, followers, and other types of transactions, and be able to be **extended after deployment** to support functions and standards that will be adopted in the future. Additionally, it should have a **secure ownership management** system to ensure the protection of valuable assets.
```

Comment:
Maybe we can break this down in bullet points?

--- 

The features that makes a smart contract an account are: 
- the ability to **verify signed messages**
- be **notified of incoming tokens**, followers, and other types of transactions,
- be able to be **extended after deployment** to support functions and standards that will be adopted in the future.

Additionally, it should have a **secure ownership management** system to ensure the protection of valuable assets.